app/src/test/java/com/github/se/orator/model/symblAi/AudioRecorderTest.kt

```diff
@@ -20,7 +21,7 @@ class AudioRecorderTest {
   @Before
   fun setUp() {
     context = mock()
-    audioRecorder = AudioRecorder(context)
+    audioRecorder = AudioRecorder(context, isOffline = false)
```

Comment:
This is a case where having a default value would have prevented you from changing outside code

Reply:
You're right ! Took care of this